source/_components/sensor.wunderground.markdown

```diff
@@ -100,19 +103,133 @@ Configuration variables:
   - **temperature_string**: Temperature text combinding Fahrenheit and Celsius
   - **temp_c**: Current temperature in Celsius
   - **temp_f**: Current temperature in Fahrenheit
+  - **temp_high_record_c**: Maximum temperature meassured in Celsius
+  - **temp_high_record_f**: Maximum temperature meassured in Fahrenheit
+  - **temp_low_record_c**: Minimal temperature meassured in Celsius
+  - **temp_low_record_f**: Minimal temperature meassured in Fahrenheit
+  - **temp_high_avg_c**: Average high for today in Celsius
+  - **temp_high_avg_f**: Average high for today in Fahrenheit
+  - **temp_low_avg_c**: Average low for today in Celsius
+  - **temp_low_avg_f**: Average low for today in Fahrenheit
+  - **temp_high_1d_c** [<sup>[1d]</sup>](#1d): Forecasted high temperature in Celsius
+  - **temp_high_1d_f** [<sup>[1d]</sup>](#1d): Forecasted high temperature in Fahrenheit
+  - **temp_low_1d_c** [<sup>[1d]</sup>](#1d): Forecasted low temperature in Celsius
+  - **temp_low_1d_f** [<sup>[1d]</sup>](#1d): Forecasted low temperature in Fahrenheit
   - **UV**: Current levels of UV radiation. See [here](https://www.wunderground.com/resources/health/uvindex.asp) for explanation.
   - **visibility_km**: Average visibility in km
   - **visibility_mi**: Average visibility in miles
   - **weather**: A human-readable text summary with picture from Wunderground.
+  - **weather_1d** [<sup>[12h]</sup>](#12h): A human-readable weather forecast using imperial units.
+  - **weather_1d_metric** [<sup>[12h]</sup>](#12h): A human-readable weather forecast using metric units.
+  - **weather_1h** [<sup>[1h]</sup>](#1h): Weather conditions in 1 hour. (e.g. "Thunderstorm" etc.)
   - **wind_degrees**: Wind degrees
   - **wind_dir**: Wind direction
   - **wind_gust_kph**: Wind gusts speed in kph
   - **wind_gust_mph**: Wind gusts speed in mph
+  - **wind_gust_1d_kph** [<sup>[1d]</sup>](#1d): Max. forecasted Wind in kph
+  - **wind_gust_1d_mph** [<sup>[1d]</sup>](#1d): Max. forecasted Wind in mph
   - **wind_kph**: Current wind speed in kph
   - **wind_mph**: Current wind speed in mph
+  - **wind_1d_kph** [<sup>[1d]</sup>](#1d): Forecasted wind speed in kph
+  - **wind_1d_mph** [<sup>[1d]</sup>](#1d): Forecasted wind speed in mph
   - **wind_string**: Text summary of current wind conditions
 
-All the conditions listed above will be updated each 5 minutes with exception of `alerts` that will be updated each 15 minutes by default.
+All the conditions listed above will be updated every 5 minutes.
+
+<p class='note'>
```

Comment:
OK, I've removed the notes and replaced them with with "plain text". 
Dissecting the forecast options to special section would be problematic as they logically belong to the monitored conditions list. But I'm open to further suggestions.